next/components/molecules/presentation/Contacts.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  ContactsSectionFragment,
+  Enum_Componentsectionscontactssection_Type,
+} from '@backend/graphql'
+import Markdown from '@components/atoms/Markdown'
+import ContactCtaCard from '@components/molecules/presentation/ContactCtaCard'
+import { isDefined } from '@utils/isDefined'
+import cx from 'classnames'
+import React from 'react'
+
+type ContactsProps = {
+  section: ContactsSectionFragment
+}
+
+const Contacts = ({ section }: ContactsProps) => {
```

Comment:
This `presentation` folder groups all card-like components (based on `CardBase` or `Row base if implemented in future).

Please do not mix `Contacs` here :D